src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -43,16 +78,21 @@ public static Query buildSearchConceptsQuery(
             true);
 
     // domain clause filters for the given domain id based on field domain_id
```

Comment:
I think we either need the SQL, or this comment, but I wouldn't have both of them. I would maybe get rid of this line and leave the SQL.